2021-09-05 17:15:05,285 - no description
2021-09-05 17:15:05,285 - dataset:ml-1m lr1:0.001 lr2:0.0003 kl:0.05 contrast:0.1 batch:1 model_func:fc_att
2021-09-05 17:15:05,285 - Using CUDA:0
2021-09-05 17:15:06,166 - Building net...
2021-09-05 17:15:06,181 - Net build finished.
2021-09-05 17:15:06,181 - User datasets loaded and saved.
2021-09-05 17:15:06,183 - Start training...
2021-09-05 17:15:06,184 - Traceback (most recent call last):
  File "trainMe0904.py", line 299, in <module>
    train()
  File "trainMe0904.py", line 202, in train
    cur_cnt = cur_cnt.to(hyper_params['device'])
AttributeError: 'int' object has no attribute 'to'

2021-09-05 17:15:06,185 - Error.
